docs/pages/visualization.md

```diff
@@ -0,0 +1,83 @@
+@page visualization Visualization Guide
+
+[TOC]
+
+# About
+
+svZeroDVisualization is a web application designed for visualizing 0D simulation results and the 0D network. It allows users to interactively explore and analyze their simulation data through an intuitive interface. 
+This application is available in the  `applications` folder.
+
+
+# Architecture
+svZeroDVisualization is built using a robust architecture that includes:
+- Frontend: Utilizes HTML, CSS, Dash, and Plotly for creating a dynamic and interactive user interface. This setup allows for effective visualization and interaction with the 0D network and simulation results.
+- Backend: Powered by a Flask application that handles the server-side logic. It leverages NetworkX for managing and visualizing the network graph and a Python code to determine network connections.
+
+# Installing Dependencies 
+1. We recommend using a virtual environment to help manage project-specific 
+dependencies and avoid conflicts with other projects.
+- Using venv:
+```bash
+python -m venv venv
+source venv/bin/activate  # On Windows use `venv\Scripts\activate`
+```
+- Using Conda:
+```bash
+conda create --name myenv python=3.12  # Replace with your desired Python version
+conda activate myenv
+```
+
+2. Install the necessary packages:
+```bash
+pysvzerod
+pandas
+matplotlib
+networkx
+dash
+plotly
+numpy
+```
+
+
+# How to Use
+1. Navigate to the `applications` folder and then into the `dirgraph_visualization` subdirectory.
+
+2. Open the dirgraph_main.py file.
```

Comment:
Do they still need to open this if the arguments are from the command line?

Reply:
Nope you're right - Deleted this and reformatted the instructions!